src/test/groovy/hudson/plugins/gradle/injection/BuildScanInjectionMavenIntegrationTest.groovy

```diff
@@ -293,14 +293,32 @@ class BuildScanInjectionMavenIntegrationTest extends BaseMavenIntegrationTest {
             accessKey = Secret.fromString("scans.gradle.com=secret")
         }
         def pipelineJob = j.createProject(WorkflowJob)
-        pipelineJob.setDefinition(new CpsFlowDefinition(simplePipeline(mavenInstallationName), false))
+        pipelineJob.setDefinition(new CpsFlowDefinition("""
+node {
+   stage('Build') {
+        node('foo') {
+            withEnv(["PATH+MAVEN=\${tool '${mavenInstallationName}'}/bin"]) {
+                writeFile file: 'pom.xml', text: '$POM_XML'
+                if (isUnix()) {
+                    sh "echo \$GRADLE_ENTERPRISE_ACCESS_KEY"
+                    sh "echo \$DEVELOCITY_ACCESS_KEY"
+                    sh "mvn package -B"
+                } else {
+                    bat "echo %GRADLE_ENTERPRISE_ACCESS_KEY%"
+                    bat "echo %DEVELOCITY_ACCESS_KEY%"
+                    bat "mvn package -B"
+                }
+            }
+        }
+   }
+}
+""", false))
 
         when:
         def build = j.buildAndAssertSuccess(pipelineJob)
 
         then:
-        j.assertLogContains("GRADLE_ENTERPRISE_ACCESS_KEY=scans.gradle.com=secret", build)
-        j.assertLogContains("DEVELOCITY_ACCESS_KEY=scans.gradle.com=secret", build)
+        j.assertLogContains("scans.gradle.com=secret", build)
```

Comment:
I would keep the assertions and instead echo `GRADLE_ENTERPRISE_ACCESS_KEY=$GRADLE_ENTERPRISE_ACCESS_KEY` in the build, same for the other env vars

Reply:
Agree, changed